label_buddy/projects/helpers.py

```diff
@@ -581,6 +585,25 @@ def project_statistics(project, user):
         return all_tasks_count, annotated_tasks, not_annotated_tasks
 
 
+# GSoC22 
+def get_audiowaveform_data(audio_file):
+
+    """
+    Return audio waveform data for the task.
+    """
+
+    json_name = audio_file.split(".")[0]
+    json_name = "media/data/" + json_name.split("audio/")[1]
+
+    os.system(f'audiowaveform -i {audio_file[1:]} -o {json_name}.json --pixels-per-second 20 --bits 8')
```

Comment:
Is is possible not to store the file in the database? Maybe you can delete the file after getting the desired information.